engwales-opinion.sty

```diff
@@ -0,0 +1,52 @@
+\ProvidesPackage{engwales-opinion}[2021-02-13 Package for pleading creation in England/Wales styles]
+
+\RequirePackage[explicit]{titlesec}
+\RequirePackage{enumitem}
+
+
+%% Formatting the main title
+\renewcommand\maketitle{%
+  \vspace*{-1.5cm}
+  \begin{center}
+    \bfseries
+    \renewcommand\arrayrulewidth{1.1pt}
+    \begin{tabular}{@{\hspace{12pt}}c@{\hspace{12pt}}}
+      \hline\\[-12pt]
+      \MakeUppercase{\@title}\\\ \\[-12pt]
+      \hline
+    \end{tabular}
+  \end{center}
+  \medskip
+}
+
+%% Formatting section and subsection titles
+\titleformat{\section}{\normalfont\bfseries}{\thesection}{1em}
+  {\vspace*{-0.25em}\underline{\MakeUppercase{#1}}\medskip}
+\titleformat{\subsection}{\normalfont\bfseries}{\thesection}{1em}
+  {\vspace*{-0.25em}\underline{#1}\medskip}
+\titleformat{\subsubsection}{\normalfont}{\thesection}{1em}
+  {\medskip\underline{#1}\medskip}
+
+%% Create intro and facts commands for supposed convience
+\newcommand{\intro}{\section*{INTRODUCTION}}
+\newcommand{\facts}{\section*{FACTS}}
```

Comment:
altho i don't think a table of contents will be needed, why do we use starred heading everywhere? also iirc `hyperref` will respect starred headings and skip creating bookmarks for them. this might not be desired.

Reply:
Table of contents is not commonly used.

Reply:
I didn't want to create numbered sections and couldn't remember how to disable it. I'll have another look today to see what the change is, or you could make a quick commit if you know the command.

Reply:
The use of `\section*` seems fine to me

Reply:
I just changed it so you can use section. I don't know if it affects the bookmarking, but that can probably be addressed later.